release/release-notes/data-prepper.change-log-2.3.0.md

```diff
@@ -0,0 +1,1796 @@
+
+* __Updates main to Data Prepper 2.4. (#2832)__
```

Comment:
This is in `main`, but not `2.4`. So something seems wrong with the changelog generation.